test/unit/unitTestRunner.js

```diff
@@ -12,34 +12,48 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import {TraceurTestRunner} from '../modular/TraceurTestRunner.js';
+import {NodeTraceurTestRunner} from '../modular/NodeTraceurTestRunner.js';
+import {BrowserTraceurTestRunner} from '../modular/BrowserTraceurTestRunner.js';
 
-export var unitTestRunner = new TraceurTestRunner({
-  patterns: [
-    'test/unit/util/*.js',
-    'test/unit/syntax/*.js',
-    'test/unit/codegeneration/*.js',
-    'test/unit/semantics/*.js',
-    'test/unit/tools/*.js',
-    'test/unit/runtime/*.js',
-    'test/unit/system/*.js',
-    'test/unit/node/*.js',
-    'test/unit/*.js'
-  ]
-});
+export let unitTestRunner;
+let chai;
+if (typeof window !== 'undefined') {
+  unitTestRunner = new BrowserTraceurTestRunner();
+  chai = window.chai;
+} else {
+  unitTestRunner = new NodeTraceurTestRunner();
+  chai = require('chai');
+}
 
 var context = unitTestRunner.getContext();
 
-export var suite = context.suite;
+export var suite = function(title, tests) {
+  if (typeof window !== 'undefined' && title.startsWith('node-only:')) {
+    return;
+  }
+  return context.suite(title, tests);
+};
+
 export var test = context.test;
 export var setup = context.setup;
 export var teardown = context.teardown;
 
-var chai = require('chai');
 export var assert = chai.assert;
 export var AssertionError = chai.AssertionError;
 
 export function assertArrayEquals(expected, actual) {
   assert.equal(JSON.stringify(actual, null, 2),
                JSON.stringify(expected, null, 2));
-}
\ No newline at end of file
+}
+
+unitTestRunner.applyOptions([
+    'test/unit/util/*.js',
```

Comment:
Array and object literals are indented 2 spaces.

https://google-styleguide.googlecode.com/svn/trunk/javascriptguide.xml?showone=Code_formatting#Code_formatting

> Multiline array initializers and object initializers are indented 2 spaces, with the braces on their own line, just like blocks.